### ISSUE TITLE
Print Coverage Total for All Projects

### ISSUE THREAD

**Issue by AnastasiyaSoyka** [ksoichiro/gradle-console-reporter#6]
This feature would be helpful when working with [GitLab CI](https://about.gitlab.com/features/gitlab-ci-cd/), which expects a single coverage value and therefore uses only the last project's coverage value as the total coverage value.

**Comment by louth** [comment]
This is achievable provided you combine your coverage tool's coverage reports into a single coverage report and then configure the console-reporter in the top level build.gradle to access it, like this:

```
consoleReporter {
    jacoco {
        reportFile = ["${buildDir}/reports/jacoco/test/jacocoTestReport.xml"]
    }
}
```

**Comment by amimas** [comment]
I also need a "Total Coverage" for all projects but can't seem to get this to work. If I run `gradlew clean build`, I automatically get the coverage of each of the Gradle sub projects' coverage.

I added a separate task that aggregates the reports from all sub projects and creates a separate single coverage report. I can view this combined report from "rootProjectDir/build/reports/jacoco/rootCoverageReport/rootCoverageReport.xml"

I configured this plugin based on the recommendation mentioned above by pointing `reportFile` to above mentioned path, but if I run `reportJacoco` task, the "Coverage summary" section still shows 0.0%
